src/status_im2/contexts/chat/photo_selector/view.cljs

```diff
@@ -18,52 +19,62 @@
   [:f>
    (fn []
      (let [safe-area (safe-area/use-safe-area)]
-       (when (or (pos? (count @selected)) selected-images)
+       (when (or (seq @selected) selected-images)
          [linear-gradient/linear-gradient
           {:colors [:black :transparent]
            :start  {:x 0 :y 1}
            :end    {:x 0 :y 0}
            :style  (style/gradient-container safe-area)}
-          [quo2/button
-           {:style    {:align-self        :stretch
-                       :margin-horizontal 20}
-            :on-press #(do
-                         (rf/dispatch [:chat.ui/clear-sending-images chat-id])
-                         (doseq [item @selected]
-                           (rf/dispatch [:chat.ui/camera-roll-pick item]))
-                         (reset! selected [])
-                         (rf/dispatch [:bottom-sheet/hide]))}
+          [quo/button
+           {:style               {:align-self        :stretch
+                                  :margin-horizontal 20}
+            :on-press            (fn []
```

Comment:
No major concern, but I think the code is more understandable when we have single-purpose functions.

You can extract the `on-press` anonymous function. 

```clojure
(defn on-press-confirm-selection
  [chat-id]
  (rf/dispatch [:chat.ui/clear-sending-images chat-id])
  (doseq [item @selected]
    (rf/dispatch [:chat.ui/camera-roll-pick item]))
  (reset! selected [])
  (rf/dispatch [:bottom-sheet/hide]))

(defn bottom-gradient
  [chat-id selected-images]
  [:f>
   (fn []
     (let [safe-area (safe-area/use-safe-area)]
       (when (or (seq @selected) selected-images)
         [linear-gradient/linear-gradient
          {:colors [:black :transparent]
           :start  {:x 0 :y 1}
           :end    {:x 0 :y 0}
           :style  (style/gradient-container safe-area)}
          [quo/button
           {:style               {:align-self        :stretch
                                  :margin-horizontal 20}
            :on-press            #(on-press-confirm-selection chat-id)
            :accessibility-label :confirm-selection}
           (i18n/label :t/confirm-selection)]])))])
```